Casks/libcblite.rb

```diff
@@ -0,0 +1,33 @@
+cask "libcblite" do
+  version "3.0.0"
+  sha256 "abd416d4753b204a1327ffd53f4b12fc5eec2419191b8b0fa55b78d79b7b23a2"
+
+  url "https://packages.couchbase.com/releases/couchbase-lite-c/#{version}/couchbase-lite-c-enterprise-#{version}-macos.zip"
+  name "libcblite"
+  desc "Couchbase Lite Libraries for C and C++ (Enterprise Edition)"
+  homepage "https://www.couchbase.com/products/lite"
+
+  livecheck do
+    url "http://appcast.couchbase.com/couchbase-lite-version.txt"
+    regex(/^\d+\.\d+\.\d+(?:-[a-z0-9]+)?$/i)
```

Comment:
The following adheres to prevailing standards for livecheck regex.
Unless the version is subject to change in format?

```suggestion
    regex(/^(\d+(?:\.\d+)+)$/i)
```

Reply:
I initially made this regex to accommodate potential beta versions (e.g. 3.0.0-beta02) but now I realize that it's not appropriate here since beta versions go to a different repo.  I'll simplify this.